homeassistant/components/miele/sensor.py

```diff
@@ -663,28 +724,25 @@ def native_value(self) -> StateType:
         """Return the state of the sensor."""
         return self.entity_description.value_fn(self.device)
 
+    @property
+    def unique_id(self) -> str:
+        """Return the unique ID of the entity."""
+        return self.entity_description.get_unique_id(self._device_id)
```

Comment:
Moved here from attribute in order to use the fn provided now in `EntityDescription`, in order to share creation of unique id with its recreation for looking up in entity registry.